.github/workflows/ssm-release.yml

```diff
@@ -0,0 +1,249 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+name: release SSM package
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'the version number to release'
+        required: true
+      sha:
+        description: 'the github sha to release'
+        required: true
+      public:
+        description: 'set true if make SSM package public'
+        required: false
+        default: 'false'
+
+env:
+  SSM_PKG_NAME: AWSDistroOTel-Collector
+  SSM_S3_BUCKET: aws-otel-collector
+  SSM_S3_BICKET_HKG: aws-otel-collector-hkg
+  SSM_S3_BICKET_BAH: aws-otel-collector-bah
+  SSM_S3_BICKET_CPT: aws-otel-collector-cpt
+  SSM_S3_BICKET_MXP: aws-otel-collector-mxp
+  SSM_S3_BICKET_CN: aws-otel-collector-cn
+  RC_S3_BUCKET: aws-otel-collector-release-candidate
+
+jobs:
+  release-ssm-package:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the sha
+        uses: actions/checkout@v2
+        # with:
```

Comment:
Are we commenting out the sha on purpose?

Reply:
It is commented for testing and I will fix it.